### PR TITLE
task/add-description-of-datetime-in-rest-api-documentation-jaia-1704

### PR DESCRIPTION
**Pull request by edsanville**
* Auto-generate a `Parameters` section that lists the parameters for each action message, along with their "doc" descriptions from the .proto file (if any)
* Added a doc to the start_time and end_time parameters for the task_packet endpoint, describing that they are Unix timestamps in microseconds.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -46,6 +46,6 @@ git submodule update --init
 
 echo "Configuring..."
 cd ${script_dir}/build/${ARCH}
-(set -x; cmake ../.. ${JAIABOT_CMAKE_FLAGS})
```
Comment (tsaubergine) [inline build.sh]: I'm curious about the motivation for this change?

**Review by tsaubergine — DISMISSED** [review]
Updates to autogenerate parameter docs and document unix timestamps looks good to me.

**Comment by michael-jaia** [comment]
Hey Ed how do you see the final product before it is deployed to the doxygen site?

**Comment by tsaubergine** [comment]
> Hey Ed how do you see the final product before it is deployed to the doxygen site?

(My middle name is Ed so I can answer it too)

Enable building documentation:
```
cd jaiabot/build/amd64
cmake . -Dbuild_doc=ON
make
```

Then 

```
jaia doc rest_api
```
or for Markdown directly
```
 jaia doc rest_api --raw
```
or to see Doxygen website open browser to file:///home/toby/opensource/jaiabot/build/amd64/src/doc/html/md_page12_rest_api.html (changing your path to whenever jaiabot is built on your system)

**Pull request merged**